rotkehlchen/chain/evm/constants.py

```diff
@@ -5,4 +5,22 @@
 MAX_BLOCKTIME_CACHE = 250  # 55 mins with 13 secs avg block time
 ZERO_ADDRESS = string_to_evm_address('0x0000000000000000000000000000000000000000')
 ETH_SPECIAL_ADDRESS = string_to_evm_address('0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE')
-GENESIS_HASH = deserialize_evm_tx_hash('0x' + '0' * 64)  # hash for transactions in genesis block
+EMPTY_64_HEX = '0x' + '0' * 64
+GENESIS_HASH = deserialize_evm_tx_hash(EMPTY_64_HEX)  # hash for transactions in genesis block
+
+EMPTY_RECEIPT = {  # Uses ethereum receipt structure
```

Comment:
This is not just an empty receipt. It's specifically made to be fake and tied to the genesis block

```suggestion

# Fake receipt with values taken from ethereum mainnet, to emulate a receipt for the genesis transactions
FAKE_GENESIS_TX_RECEIPT = {
```